docker-compose.yml

```diff
@@ -12,18 +12,22 @@ services:
       - '10000s'
       - --write-timeout
       - '10000s'
-    image: semitechnologies/weaviate:1.9.0
-    restart: on-failure:0
+    image: semitechnologies/weaviate:preview-automatically-request-properties-that-are-needed-for-groupby-e98008c
+    restart: on-failure:1
     ports:
-     - "9999:8080"
+     - "8080:8080"
+     - "50051:50051"
     environment:
       QUERY_DEFAULTS_LIMIT: 25
+      QUERY_MAXIMUM_RESULTS: 10000
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
       PERSISTENCE_DATA_PATH: "./data"
       DEFAULT_VECTORIZER_MODULE: text2vec-transformers
       ENABLE_MODULES: text2vec-transformers
+      CLUSTER_HOSTNAME: 'node1'
+      AUTOSCHEMA_ENABLED: 'false'
       TRANSFORMERS_INFERENCE_API: http://t2v-transformers:8080
   t2v-transformers:
-    image: semitechnologies/transformers-inference:sentence-transformers-msmarco-distilroberta-base-v2
```

Comment:
Is it necessary to change the model?

Reply:
I changed it according to the suggestions by Tommy on Slack, since the model before is now deprecated.